src/components/v2/charts/InterestRateChart/index.tsx

```diff
@@ -0,0 +1,135 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import {
+  ResponsiveContainer,
+  YAxis,
+  XAxis,
+  CartesianGrid,
+  LineChart,
+  Line,
+  ReferenceLine,
+  Tooltip,
+} from 'recharts';
+
+import { useTranslation } from 'translation';
+import { formatToReadablePercentage } from 'utilities/common';
+import TooltipContent from '../TooltipContent';
+import { useStyles as useSharedStyles } from '../styles';
+import { useStyles as useLocalStyles } from './styles';
+
+export interface IInterestRateItem {
+  utilizationRate: number;
+  borrowApyPercentage: number;
+  supplyApyPercentage: number;
+}
+
+export interface IInterestRateChartProps {
+  data: IInterestRateItem[];
+  currentUtilizationRate: number;
+  className?: string;
+}
+
+export const InterestRateChart: React.FC<IInterestRateChartProps> = ({
+  className,
+  currentUtilizationRate,
+  data,
+}) => {
+  const sharedStyles = useSharedStyles();
+  const localStyles = useLocalStyles();
+  const styles = {
+    ...sharedStyles,
```

Comment:
In this case it would indeed be a feature. The idea is that `sharedStyles` represent the base styles used by all components importing it. If one of the component wishes to override of the element, it can do so by redeclaring it inside its own local styles. In practice, we haven't had to do that so far, but it's an option.

That said, separating them works for me too. If you feel it's clearer to you, I'll make the change 👍